completions/rsync

```diff
@@ -1,5 +1,35 @@
 # bash completion for rsync                                -*- shell-script -*-
 
+_comp_cmd_rsync__vercomp()
+{
+    if [[ "$1" == "$2" ]]; then
```

Comment:
```suggestion
    if [[ $1 == "$2" ]]; then
```

Actually, the `==` operator of the conditional command `[[ ]]` is not symmetric. The right-hand side of `==` in the conditional command `[[ ]]` is considered a glob pattern so needs to be quoted for the literal comparison. But the left-hand side doesn't need to be quoted.